R/pkg/R/stats.R

```diff
@@ -52,22 +52,17 @@ setMethod("crosstab",
             collect(dataFrame(sct))
           })
 
-#' Calculate the sample covariance of two numerical columns of a SparkDataFrame.
```

Comment:
I see. in that case can we add this in the `@details` for `cov`
I feel like this has bits of info that could be useful `Calculate the sample covariance of two numerical columns of a SparkDataFrame` - say, *numerical* columns of *one* SparkDataFrame, as supposed to `cov(df, df$name, df2$bar)`